examples/nodejs/aws/eventbridge/deploy-and-run-cli.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+set -e
+set -x
+
+# Check if .env file exists
+if [ ! -f .env ]; then
+    echo ".env file not found. See README.md for more information: https://github.com/momentohq/client-sdk-javascript/tree/main/examples/nodejs/aws/eventbridge/webapp/README.md"
+    exit 1
+fi
+
+# Load environment variables from .env.development file
+set -a
+source .env
+set +a
+
+# Check if environment variables are loaded
+if [ -z "$MOMENTO_API_KEY" ] || [ -z "$AWS_ACCESS_KEY_ID" ] || [ -z "$AWS_SECRET_ACCESS_KEY" ] || [ -z "$AWS_REGION" ]; then
+    echo "Error: Environment variables not loaded from .env file."
+    exit 1
+fi
+
+if [ -z "$AWS_ACCESS_KEY_ID" ] || [ -z "$AWS_SECRET_ACCESS_KEY" ] || [ -z "$AWS_REGION" ]; then
+    echo "Error: Environment variables not loaded from .env file."
+    exit 1
+fi
+
+# Run the deploy script with environment variables
+cd infrastructure
+npm install
+npm run build
+npm run deploy -- --parameters MomentoApiKey="$MOMENTO_API_KEY" --parameters MomentoApiEndpoint="$MOMENTO_API_ENDPOINT"
+
+# cd back to the root directory
+cd ..
+
+# Set your DynamoDB table name
+TABLE_NAME="weather-stats-demo"
+
+# Set your item attributes
+LOCATION="Phoenix"
+MAX_TEMP="80"
+MIN_TEMP="60"
+PRECIPITATION="90"
+TTL="300" # 5 minutes
+
+# Set Momento Cache and Topic Name
+CACHE_NAME="momento-eventbridge-cache"
+TOPIC_NAME="momento-eventbridge-topic"
+
+## Install Momento CLI
+brew tap momentohq/tap
+brew install momento-cli
+brew upgrade momento-cli
```

Comment:
FYI - I would probably advise to never put brew commands in a script that you are asking users to run :) it's pretty easy to completely break your brew install so you should let them deal with that on their own. not relevant here because of another comment i'm going to make about doing this script in TS though :)